source/sass/components/_form.scss

```diff
@@ -39,12 +39,27 @@
       display: grid;
       grid-template-columns: 0.3fr 0.7fr;
 
-      .form-item:nth-of-type(2) {
-        input {
-          border-left: 0;
+      .city-input-container {
+        .form-item {
+          input {
+            border-left: 0;
+          }
+        }
+      }
+
+      &--full-width {
+        grid-template-columns: 1fr;
+        .city-input-container {
+          .form-item {
+            input {
```

Comment:
Since in the original 2 column format the border-left is set to 0, we are setting the " input field border to match the rest of the fields when in the full width format.